corehq/apps/reports/static/reports/js/config.dataTables.bootstrap.js

```diff
@@ -1,5 +1,5 @@
-hqDefine("reports/js/config.dataTables.bootstrap", function() {
-
+/* global hqDefine, jQuery */
+hqDefine("reports/js/config.dataTables.bootstrap", ['jquery', 'datatables.bootstrap'], function($) {
```

Comment:
can this be changed to `define` instead of `hqDefine`?

Reply:
I think we talked about this, but for posterity: this particular `hqDefine` can't be changed because it's still used in a non-RequireJS context, but in general I'm on board with switching modules over to `define` once they're only used in RequireJS contexts.